scripts/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+echo "starting..."
+
+set -o errexit
+
+TAG='janis:local'
+echo "building docker image..."
+docker build --tag "$TAG" .
+echo "running docker image..."
+docker run \
+    --rm \
+    --name janis \
+    --tty --interactive \
+    --volume "$PWD/dist:/app/_dist" \
+    --volume "$PWD/src:/app/src" \
+    --volume "$PWD/public:/app/public" \
+    --volume "$PWD/package.json:/app/package.json" \
+    --volume "$PWD/static.config.js:/app/static.config.js" \
+    --volume "$PWD/.babelrc:/app/.babelrc" \
+    --volume "$PWD/yarn.lock:/app/yarn.lock" \
+    --env "CMS_API=http://$(ifconfig en0 | awk '$1 == "inet" {print $2}'):8000/api/graphql/" \
```

Comment:
You should run the command for getting IP just once

Untested but something like
```bash
HOST_IP=$(ifconfig en0 | awk '$1 == "inet" {print $2}')
docker run \
...
  --env "CMS_API=http://$HOST_IP:8000/api/graphql/" \
  --env "CMS_MEDIA=http://$HOST_IP:8000/media/" \
```